site/content/en/latest/user/private-key-provider.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Using Envoy Private Key Provider for TLS termination"
+---
+
+The Envoy Private Key Provider (https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#extensions-transport-sockets-tls-v3-privatekeyprovider) allows implementing different provider which can provide the private key operation acceleration through the hardware or private key protection through the hardware.
+
+Today, there are two private key providers were implemented as Envoy contrib-extension:
+ CryptoMB in Envoy 1.20 release (https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/private_key_providers/cryptomb/v3alpha/cryptomb.proto )
+QAT in Envoy 1.24 release ( https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/private_key_providers/qat/v3alpha/qat.proto#extensions-private-key-providers-qat-v3alpha-qatprivatekeymethodconfig )
+
+Both of them are used to accelerate the TLS handshake through the hardware capabilities.
+
+This guide will walk through the steps required to configure TLS Terminate mode for TCP traffic and same time using the Envoy Private Key Provider to accelerate the TLS handshake.
+
+## Prerequisites
+
```

Comment:
as a user reading this doc, would prefer if this section read as 
* how do I know if my HW has this capability or not
* how do I turn on the HW/SW capability